content/en/docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -8,7 +8,7 @@ weight: 30
 
 {{% capture overview %}}
 
-<img src="https://raw.githubusercontent.com/cncf/artwork/master/kubernetes/certified-kubernetes/versionless/color/certified-kubernetes-color.png" align="right" width="150px">**kubeadm** helps you bootstrap a minimum viable Kubernetes cluster that conforms to best practices.  With kubeadm, your cluster should pass [Kubernetes Conformance tests](https://kubernetes.io/blog/2017/10/software-conformance-certification). Kubeadm also supports other cluster 
+<img src="https://github.com/cncf/artwork/blob/master/projects/kubernetes/certified-kubernetes/1.14/color/certified-kubernetes-1.14-color.png" align="right" width="150px">**kubeadm** helps you bootstrap a minimum viable Kubernetes cluster that conforms to best practices.  With kubeadm, your cluster should pass [Kubernetes Conformance tests](https://kubernetes.io/blog/2017/10/software-conformance-certification). Kubeadm also supports other cluster 
```

Comment:
It feels like we need a way to have Hugo interpolate the version number in, rather than use `1.14` forever.

(maybe a new issue for that? what do people think?)

Reply:
The [deploy preview](https://deploy-preview-14486--kubernetes-io-master-staging.netlify.com/docs/setup/independent/install-kubeadm/) still seems to have the issue BTW

Reply:
@sftim I am able to view it in the patch changes though when i preview the file

Reply:
The image url is not actually the url to png, it's a github url